### ISSUE TITLE
Prompts with type:markdown are missing the "body" field

### ISSUE THREAD

**Issue by calebhailey** [sensu/catalog-api#30]
Current behavior: 

![IMG_0359](https://user-images.githubusercontent.com/141373/155445453-f6286b30-fa80-44a6-9585-f1a2c7cc4ece.PNG)

At first I thought this might be a [web UI issue](https://github.com/sensu/sensu-enterprise-go/issues/2072) but then @jamesdphillips noticed that the data is missing in the API. 

Integration: https://github.com/sensu/catalog/blob/main/integrations/slack/slack-alerts/sensu-integration.yaml#L20-L29 

Example build: 

```
curl -s https://catalog.sensu.io/3ce33e1e326cd8b8bb8813286ae3b7918031f47f572850f8aba5ec458322dc74/v1/mattermost/mattermost-alerts.json | jq .prompts[1]
```

Output: 

```json
{
  "type": "markdown",
  "title": "",
  "name": "",
  "input": null
}
```

**Comment by jamesdphillips** [comment]
Closed by #32

**Comment by amdprophet** [comment]
Closed by #32.
